uwblueprint/focus-on-nature
# feat: create camper edit api

## Notion ticket link
[Camper Edit API](https://uwblueprintexecs.notion.site/Camper-Edit-API-0bca94a415f549bd88e447a6b587ba3f)


## Implementation description
* added a UpdateCamperDTO in types.ts 
* created a camper service interface and class 


## Steps to test
1. Make a PUT request on Postman with the following endpoint http://localhost:5000/camper/[camperId]
2. Enter field key and value that needs to be updated 
3. Submit PUT request 
4. Check MongoDB to confirm change populated 


## What should reviewers focus on?
* Are all the necessary components present? 


## Checklist
